### PR TITLE
Add SplitUHC

### PR DESCRIPTION
**Pull request by AjuMeX**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] No changes were made to the file's formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [x] My image is a `png` file.
* [x] I uploaded my image to the `logos` directory.
* [x] My image has a transparent background and is square (1:1 aspect ratio).
* [x] My image is `512` pixels in width and height.
* [x] My image's file name matches its mapping id.

### CODE REVIEW

**Review by jacobk999 — CHANGES_REQUESTED** [review]
Make this a png

**Comment by AjuMeX** [comment]
> Make this a png

Ready 👍

File: servers.json [hunk 1]
```diff
@@ -14,6 +14,14 @@
         "addresses": [
             "menastro.gg"
         ]
+    },
+	{
+        "id": "split",
```
Comment (jacobk999) [inline servers.json:19]:
```suggestion
        "id": "splituhc",
```

File: servers.json [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
+        "name": "SplitUHC",
+        "addresses": [
+            "mc.splituhc.tk",
+	    "splituhc.tk"
+        ]
```
Comment (jacobk999) [inline servers.json:24]:
```suggestion
        "addresses": [
	    "splituhc.tk"
        ]
```

**Review by jacobk999 — CHANGES_REQUESTED** [review]

File: servers.json [hunk 3]
```diff
@@ -14,6 +14,13 @@
         "addresses": [
             "menastro.gg"
         ]
+    },
```
Comment (jacobk999) [inline servers.json:17]: Your server logo is not a real `png`.

**Review by jacobk999 — CHANGES_REQUESTED** [review]

File: servers.json [hunk 4]
```diff
@@ -14,6 +14,13 @@
...
+	{
```
Comment (GrowlyX) [inline servers.json:18]: Please remove this indentation.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Pull request closed without merging**